### PR TITLE
redirect www-ccd to manage-case on perftest

### PR DESCRIPTION
**Pull request by thomast1906**
### Change description ###
redirect www-ccd to manage-case on perftest

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: k8s/namespaces/ccd/ccd-api-gateway-web/perftest-redirect.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# this is to apply permanent redirect on www-ccd.perftest.platform.hmcts.net
+---
+apiVersion: extensions/v1beta1
```
Comment (timja) [inline k8s/namespaces/ccd/ccd-api-gateway-web/perftest-redirect.yaml:3]: could you update the latest ingress api? this is deprecated for removal

**Review by timja — APPROVED** [review]

File: k8s/namespaces/ccd/ccd-api-gateway-web/perftest-redirect.yaml (hunk 1 above)
Comment (thomast1906) [reply k8s/namespaces/ccd/ccd-api-gateway-web/perftest-redirect.yaml]: cheers - now updated

**Review by timja — DISMISSED** [review]
Just double checking you’ve confirmed that the cluster CFT is on supports this version and no other changes required?

would check but on phone

File: k8s/namespaces/ccd/ccd-api-gateway-web/perftest-redirect.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# this is to apply permanent redirect on www-ccd.perftest.platform.hmcts.net
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: traefik
+    traefik.ingress.kubernetes.io/redirect-permanent: "true"
+    traefik.ingress.kubernetes.io/redirect-regex: ^https://www-ccd.perftest.platform.hmcts.net/(.*)
```
Comment (danlysiak) [inline k8s/namespaces/ccd/ccd-api-gateway-web/perftest-redirect.yaml:9]:
```suggestion
    traefik.ingress.kubernetes.io/redirect-regex: ^https://www-ccd.perftest.platform.hmcts.net(.*)
```
With the `/`, will it work for users who just try going to `https://www-ccd.perftest.platform.hmcts.net`?

**Review by MSancaktutar — CHANGES_REQUESTED** [review]
wouldn't this be on case-management-web file?

**Review by MSancaktutar — DISMISSED** [review]
lgtm

File: k8s/namespaces/ccd/ccd-case-management-web/perftest-redirect.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# this is to apply permanent redirect on www-ccd.perftest.platform.hmcts.net
+---
+apiVersion: extensions/v1beta1
```
Comment (timja) [inline k8s/namespaces/ccd/ccd-case-management-web/perftest-redirect.yaml:3]: You may want to have this not kustomized? Guessing that’s why checks are failing

Comment (thomast1906) [reply k8s/namespaces/ccd/ccd-case-management-web/perftest-redirect.yaml]: cheers, trying to do this in a meeting didn't return the result :D

**Review by MSancaktutar — DISMISSED** [review]

**Review by timja — DISMISSED** [review]

File: k8s/perftest/common/ccd/ccd-case-management-web-redirect.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: traefik
+    traefik.ingress.kubernetes.io/redirect-permanent: "true"
+    traefik.ingress.kubernetes.io/redirect-regex: ^https://www-ccd.perftest.platform.hmcts.net(.*)
+    traefik.ingress.kubernetes.io/redirect-replacement: https://manage-case.perftest.platform.hmcts.net
+  name: ccd-case-management-web-ingress
```
Comment (timja) [inline k8s/perftest/common/ccd/ccd-case-management-web-redirect.yaml:11]:
```suggestion
  name: ccd-case-management-web-redirect
```

**Review by timja — APPROVED** [review]

**Review by MSancaktutar — APPROVED** [review]

**Pull request merged**